spec/models/hyrax/permission_template_spec.rb

```diff
@@ -1,271 +1,249 @@
 # frozen_string_literal: true
-RSpec.describe Hyrax::PermissionTemplate, :clean_repo do
-  let(:admin_set) { create(:admin_set) }
-  let(:collection) { create(:collection) }
-  let(:permission_template) { described_class.new(attributes) }
+RSpec.describe Hyrax::PermissionTemplate, valkyrie_adapter: :test_adapter do
+  subject(:permission_template) { described_class.new(attributes) }
   let(:attributes) { { source_id: admin_set.id } }
-
-  subject { permission_template }
+  let(:admin_set) { FactoryBot.valkyrie_create(:hyrax_admin_set) }
 
   it { is_expected.to have_many(:available_workflows).dependent(:destroy) }
   it { is_expected.to have_one(:active_workflow).conditions(active: true).dependent(nil) }
   it { is_expected.to have_many(:access_grants).dependent(:destroy) }
 
-  describe 'factories' do
```

Comment:
Always a code smell when you _must_ test a factory's setup; Usually indicative of a series of architectural challenges. I like that we had them to verify things, but am happy to see them go away, as it shows we're moving down a better path.